src/backend/marsha/core/tests/test_views_lti_video.py

```diff
@@ -1256,7 +1256,7 @@ def test_views_lti_video_static_base_url(self, mock_get_consumer_site, mock_veri
         response = self.client.post("/lti/videos/{!s}".format(video.pk), data)
         self.assertEqual(response.status_code, 200)
         self.assertContains(response, "<html>")
-        self.assertContains(response, '<meta name="public-path" value="/static/js/" />')
+        self.assertContains(response, '<meta name="public-path" value="/static/js/build/" />')
```

Comment:
I should you should add a line break.
https://app.circleci.com/pipelines/github/openfun/marsha/1733/workflows/e96f4f80-f4c0-4e9d-8524-4463fe09fe4b/jobs/73248